docs/tutorials/stateless-application/hello-minikube.md

```diff
@@ -381,6 +381,12 @@ kubectl delete service hello-node
 kubectl delete deployment hello-node
 ```
 
+Optionally, remove the Docker images created:
```

Comment:
Since you use the -f option, it would be a little more accurate to say:
Optionally, force removal of the Docker images created:

Reply:
great suggestion 👍